src/bot.ts

```diff
@@ -1,5 +1,5 @@
 import { strict as assert } from 'assert'
-import TelegramBot, { Message } from 'node-telegram-bot-api'
+import TelegramBot from 'node-telegram-bot-api'
```

Comment:
Not that it matters, but why?

Reply:
Cause the linter was complaining to me! 
For some reason, he thought that I was not using Message!

Reply:
That sounds like linter didn't get that we using TS

Reply:
It suggested me to import it, then I do it, and it says that it's unused :(

Reply:
Does it happen to you too??

![image](https://user-images.githubusercontent.com/2352112/68488487-6c3a9900-0245-11ea-970b-1ca160ac8263.png)

Reply:
@anxolin works fine for me, what version TS is your VS using? it may be that your global is interrupting the workstation one or vice versa. I'm on `3.6.3`

Reply:
Thanks for the tip!

I made some tests the version and didn't work, I saw it was also failing in travis so there might be a issue within the project. 

I made a temporary workaround and created this issue to address these problems independently: https://github.com/gnosis/dex-telegram/issues/14